activerecord/lib/active_record/persistence.rb

```diff
@@ -121,8 +121,8 @@ def persisted?
     #
     # Attributes marked as readonly are silently ignored if the record is
     # being updated.
-    def save(*args)
-      create_or_update(*args)
+    def save(*args, &block)
```

Comment:
This is the public method and like @matthewd pointed it is not good to add a block to this method. Since we need it can we at least remove `&block` from the method signature on rdoc using the `:args:` directive?